pkg/network/common/common.go

```diff
@@ -69,6 +70,7 @@ func ParseNetworkInfo(clusterNetwork []networkapi.ClusterNetworkEntry, serviceNe
 	return &NetworkInfo{
 		ClusterNetworks: cns,
 		ServiceNetwork:  sn,
+		VXLANPort:       *vxlanPort,
```

Comment:
Doesn't this need to deal with vxlanPort maybe being nil, and filling in the default value if so?

If it is definitely non-nil and non-0 in all callers at this point then there's no reason to pass it as a pointer

Reply:
@danwinship because the ClusterNetworkObject gets ParseNetworkInfo called on it I passed vxlanPort as a pointer, I added sanity checks for nil